docs/modules/ROOT/pages/how-tos/metadata/sboms.adoc

```diff
@@ -1 +1,152 @@
-= Inspecting SBOMs
\ No newline at end of file
+= Inspecting SBOMs
+
+== Software bill of materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. {ProductName} provides link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[build SBOMs], which list all the software libraries that a component uses. Those libraries can enable specific functionality or facilitate development. 
+
+You can use an SBOM to better understand the composition of your software. In particular, you can analyze your SBOM to determine if your application has any vulnerable dependencies. Furthermore, to comply with cybersecurity regulations, you might need to provide your customers with the SBOM for your application's components. For example, in the United States, the link:https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/[Executive Order on Improving the Nation's Cybersecurity] requires vendors to provide SBOMs, and in Europe, the link:https://www.cisa.gov/sites/default/files/2023-09/EU%20Commission%20SBOM%20Work_508c.pdf[Cyber Resilience Act] does the same.
```

Comment:
> Furthermore, to comply with cybersecurity regulations, you might need to provide your customers with the SBOM for your application's components

It feels like the wrong abstraction to discuss customers here as users of the Konflux project would likely not have customers. Organizations that use the Konflux codebase might have customers... 

SBOMs are important to increase transparency into the supply chain, but that doesn't mean that everyone will start to produce them. Maybe we can just link to these types of regulations to highlight the additional driver/need to produce them and not talk to specifics about needing to provide SBOMs?

Reply:
What about just "...your *organization* might need to provide *its* customers..."

If we expect enterprises to use Konflux, then certainly an appeal of SBOMs from Konflux is for those enterprises to provide them to customers, to satisfy regulations.

Reply:
you can simplify it further that they may need to be provided to consumers.

Reply:
One thing ibm/rh hit us writers over the head with is that we should avoid the passive tense. Unless you've got a strong objection to it, I'm gonna use that wording: "your organization might need to provide its customers."